src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -984,36 +922,45 @@ public float CalculateSpeed()
 
     public float CalculateHitpoints()
     {
-        var maxHitpoints = Membrane.Hitpoints +
-            (Rigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER);
+        var maxHitpoints = CurrentSpecies.MembraneType.Hitpoints +
+            (CurrentSpecies.MembraneRigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER);
 
         return maxHitpoints;
     }
 
     /// <summary>
     ///   Returns the cost of the organelle that is about to be placed
     /// </summary>
-    public float CalculateCurrentOrganelleCost()
+    public int MutationPointsAfterOrganelleAddition()
     {
+        if (MovingOrganelle != null)
+        {
+            return Constants.BASE_MUTATION_POINTS - Constants.ORGANELLE_MOVE_COST;
+        }
+
         if (string.IsNullOrEmpty(ActiveActionName) || !ShowHover)
-            return 0;
+            return Constants.BASE_MUTATION_POINTS;
 
-        var cost = SimulationParameters.Instance.GetOrganelleType(ActiveActionName).MPCost;
+        var prospectiveSpecies = (MicrobeSpecies)CurrentSpecies.Clone();
+        GetMouseHex(out int q, out int r);
+        var organelle = new OrganelleTemplate(GetOrganelleDefinition(ActiveActionName),
+            new Hex(q, r), 0);
 
-        switch (Symmetry)
-        {
-            case MicrobeSymmetry.XAxisSymmetry:
-                cost *= 2;
-                break;
-            case MicrobeSymmetry.FourWaySymmetry:
-                cost *= 4;
-                break;
-            case MicrobeSymmetry.SixWaySymmetry:
-                cost *= 6;
-                break;
-        }
+        // This placement might be invalid, but that doesn't matter for measurment purposes
+        prospectiveSpecies.Organelles.Organelles.Add(organelle);
+
+        return MutationPointsAfterChange(prospectiveSpecies);
+    }
 
-        return cost;
+    public void UpdateGUI()
+    {
+        gui.SetSpeciesInfo(NewName,
+            CurrentSpecies.MembraneType,
+            CurrentSpecies.Colour,
+            CurrentSpecies.MembraneRigidity);
```

Comment:
Please don't use the "chop" style for method parameters, use the wrap style instead (ie. fill as many parameters on each line as fits within the width limit).